integration/src/ethereum/ethereum.ts

```diff
@@ -119,6 +119,7 @@ export function ethereumTests(get: () => { wallet: core.HDWallet; info: core.HDW
       "ethSignTx() - ETH",
       async () => {
         if (!wallet) return;
+        if (wallet.getVendor() === 'MetaMask') return;
```

Comment:
Methods `ethVerifyMessage` and `ethSignTx` are unsupported for MetaMask wallet

Reply:
I believe there is a function call on the wallet to check is a wallet supports signing like this and you should use that rather than checking vendor name.

Reply:
See https://github.com/shapeshift/hdwallet/blob/9d1e0b254c29925d8d5e6851097270d17da4146e/packages/hdwallet-metamask/src/metamask.ts#L110